<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddToTask</class>
 <widget class="QDialog" name="AddToTask">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>191</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>75</red>
        <green>85</green>
        <blue>85</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>112</red>
        <green>127</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>93</red>
        <green>106</green>
        <blue>106</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>37</red>
        <green>42</green>
        <blue>42</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>50</red>
        <green>57</green>
        <blue>57</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>74</red>
        <green>82</green>
        <blue>85</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>75</red>
        <green>85</green>
        <blue>85</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>113</red>
        <green>126</green>
        <blue>130</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Highlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>76</red>
        <green>85</green>
        <blue>78</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>37</red>
        <green>42</green>
        <blue>42</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="PlaceholderText">
      <brush brushstyle="SolidPattern">
       <color alpha="127">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Accent">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>75</red>
        <green>85</green>
        <blue>85</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>112</red>
        <green>127</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>93</red>
        <green>106</green>
        <blue>106</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>37</red>
        <green>42</green>
        <blue>42</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>50</red>
        <green>57</green>
        <blue>57</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>74</red>
        <green>82</green>
        <blue>85</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>75</red>
        <green>85</green>
        <blue>85</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>113</red>
        <green>126</green>
        <blue>130</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Highlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>76</red>
        <green>85</green>
        <blue>78</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>37</red>
        <green>42</green>
        <blue>42</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="PlaceholderText">
      <brush brushstyle="SolidPattern">
       <color alpha="127">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Accent">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>37</red>
        <green>42</green>
        <blue>42</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>75</red>
        <green>85</green>
        <blue>85</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>112</red>
        <green>127</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>93</red>
        <green>106</green>
        <blue>106</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>37</red>
        <green>42</green>
        <blue>42</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>50</red>
        <green>57</green>
        <blue>57</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>37</red>
        <green>42</green>
        <blue>42</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>37</red>
        <green>42</green>
        <blue>42</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>75</red>
        <green>85</green>
        <blue>85</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>75</red>
        <green>85</green>
        <blue>85</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>113</red>
        <green>126</green>
        <blue>130</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>75</red>
        <green>85</green>
        <blue>85</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="PlaceholderText">
      <brush brushstyle="SolidPattern">
       <color alpha="127">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Accent">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="font">
   <font>
    <family>Sitka</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="modelBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka</family>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="modelLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Model: </string>
   </property>
  </widget>
  <widget class="QLabel" name="staffLabel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>40</y>
     <width>49</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Staff: </string>
   </property>
  </widget>
  <widget class="QComboBox" name="staffBox">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>40</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka</family>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="addBtn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka</family>
    </font>
   </property>
   <property name="text">
    <string>Assign</string>
   </property>
  </widget>
  <widget class="QLabel" name="startDateLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka</family>
    </font>
   </property>
   <property name="text">
    <string>Start date:</string>
   </property>
  </widget>
  <widget class="QLabel" name="deadlineLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka</family>
    </font>
   </property>
   <property name="text">
    <string>Deadline: </string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka</family>
    </font>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="Header">
   <property name="geometry">
    <rect>
     <x>-40</x>
     <y>10</y>
     <width>481</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:orange;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>TASK ASSIGNMENT</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
